### PR TITLE
MULE-9305: Remove "ESB" occurrences from logs and splash

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: core/src/test/resources/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -12,8 +12,7 @@ Implementation-Vendor-Id: org.mule.patterns
 Implementation-Vendor: MuleSoft, Inc.
 Build-Date: 2012-Apr-13 16:46:57
 Build-Revision: 24242
-Description: Mule ESB and Integration Platform
-Dev-List-Email: mule-esb@mulesoft.com
```
Comment (afelisatti) [inline core/src/test/resources/META-INF/MANIFEST.MF]: Are you sure this isn't require somehow? Same with the mailing list in the pom.

Comment (elrodro83) [inline core/src/test/resources/META-INF/MANIFEST.MF]: I just removed the code that used this entry. Other than that this is a custom field.
As for the pom entry, it is entirely optional, and used only for the site generation through maven.

File: core/src/main/java/org/mule/DefaultMuleMessage.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
  */
 package org.mule;
 
-import static org.mule.util.SystemUtils.LINE_SEPARATOR;
+import static org.apache.commons.lang.SystemUtils.LINE_SEPARATOR;
```
Comment (afelisatti) [inline core/src/main/java/org/mule/DefaultMuleMessage.java]: Was this a mistake?

Comment (elrodro83) [inline core/src/main/java/org/mule/DefaultMuleMessage.java]: Seems to be, i'll change it back.

**Pull request merged**